newsletter/admin.py

```diff
@@ -0,0 +1,26 @@
+from django.conf import settings
+from django.contrib import admin
+
+from . import models
+
+
+@admin.register(models.Subscription)
+class SubscriptionAdmin(admin.ModelAdmin):
+    """
+    Admin for subscription model
+    """
+
+    list_display = [
+        "email",
+        "created_at",
+    ] or settings.NEWSLETTER_ADMIN_LIST_DISPLAY
+
+    readonly_fields = ["first_name", "last_name", "email"] or settings.NEWSLETTER_ADMIN_READONLY_FIELDS
+
+    fieldsets = [
+        (None, {"fields": ["first_name", "last_name", "email"]}),
+        (
+            "Meta Data",
+            {"classes": ("collapse",), "fields": ["created_at", "updated_at"]},
+        ),
+    ] or settings.NEWSLETTER_ADMIN_FIELDSETS
```

Comment:
Won't these always evaluate to the first items? What's the `or settings...` bit for here?

Reply:
For the time being them evaluating as the first items is fine (as it's still early days). The `or` is so that eventually you will be able to override the list if you make any adjustments to the fields

Reply:
The idea at the moment is just to get them in a state where we can use the base fields/settings in a project and use the app as we would normally

Reply:
When these are included in a Django project, it'll allow us to override the defaults with values set in `settings`.

Reply:
I believe you'd have to do something like this if you want to use defaults that can be overriden by settings. As it stands, the settings variables won't replace the defaults you've specified even if they exist